src/myr/Myr.js

```diff
@@ -1047,6 +1053,164 @@ class Myr {
         return outerElId;
     }
 
+    setSeed = (seed = 0) => {
+        if (seed === 0){
```

Comment:
If passing nothing to setSeed should generate a random seed, then you should not pass a default argument and should instead check if seed is undefined.

Reply:
What happens if you pass in a negative? Or a string? Or a JSON object?